### PR TITLE
August newsletter

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

File: content/blog/2021-08-31-ropensci-news-digest-august-2021/index.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+
+And you can also discover contributions by community members about rOpenSci packages and peer review:
+
+* Andrea Gilardi's talk [**osmextract: An R package to download, convert, and import large OpenStreetMap datasets**](https://youtu.be/8PMF7cBBH7k?t=3664) (rOpenSci-reviewed [osmextract package](https://docs.ropensci.org/osmextract)).
+
+* Michael Mahoney's talk [**Virtual Environments: Using R as a Frontend for 3D Rendering of Digital Landscapes**](https://youtu.be/tbt8ZsHm5eA?t=2235) (rOpenSci-reviewed [terrainr package](https://docs.ropensci.org/terrainr/)).
+
+* Lluís Revilla Sancho's talk [**Packages submission and reviews; how does it work?**](https://youtu.be/2B7H8WJOX40?t=3352)
+
+* Daniel Vartanian's talk [**mctq: An R Package for the Munich ChronoType Questionnaire**]() ([mctq package](https://github.com/gipsousp/mctq) currently [under review](https://github.com/ropensci/software-review/issues/434)). NO LINK AS OF NOW
```
[the remark this answers is not in the document]
Comment (maelle) [reply content/blog/2021-08-31-ropensci-news-digest-august-2021/index.md:49]: Removing the item for now and will re-add if the link becomes available.

File: content/blog/2021-08-31-ropensci-news-digest-august-2021/index.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+
+Recordings of useR! 2021 are now available on the conference website.
+
+You can watch contributions by rOpenSci staff:
+
+* Jeroen Ooms' keynote talk about the [**R-universe project**](https://www.youtube.com/watch?v=8cv2qsZ_xZw&list=PL4IzsxWztPdmHoJwIVa4um44w2GMjctmP&index=8) ([slidedeck](https://jeroen.github.io/user2021/#1)).
+
+* Stefanie Butland's talk [**rOpenSci's Model for Managing a Federated Open Source Software Community**](https://youtu.be/2B7H8WJOX40?t=1048) ([slidedeck](https://docs.google.com/presentation/d/1W4DepLd6FvunkGsTloyn-be_X5jI7dS-gAVtpsJb6iM)).
+
+* Mark Padgham's talk on [**autotest**, a package for automatic testing of packages](https://youtu.be/-PsaqMWfUIg?t=28).
```
[the remark this answers is not in the document]
Comment (maelle) [reply content/blog/2021-08-31-ropensci-news-digest-august-2021/index.md:37]: I prefer putting the whole thing in bold as autotest on its own might not mean much for readers.

File: content/blog/2021-08-31-ropensci-news-digest-august-2021/index.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+
+* Lluís Revilla Sancho's talk [**Packages submission and reviews; how does it work?**](https://youtu.be/2B7H8WJOX40?t=3352)
+
+* Daniel Vartanian's talk [**mctq: An R Package for the Munich ChronoType Questionnaire**]() ([mctq package](https://github.com/gipsousp/mctq) currently [under review](https://github.com/ropensci/software-review/issues/434)). NO LINK AS OF NOW
+
+Thanks to the volunteers team behind useR! 2021 for their work before, during and after the conference!
+
+
+### Recordings of BIRD TALKS BLURB BY STEFFI
+
```
[the remark this answers is not in the document]
Comment (maelle) [reply content/blog/2021-08-31-ropensci-news-digest-august-2021/index.md:55]: Added but note to self: ADD LINK!!!

File: content/blog/2021-08-31-ropensci-news-digest-august-2021/index.md [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+## Call for maintainers
+
+<!--IF CALL
+* [our guidance on _Changing package maintainers_](https://devguide.ropensci.org/changing-maintainers.html)
+* [our _Package Curation Policy_](https://devguide.ropensci.org/curationpolicy.html)
+
+IF NO CALL
+There's no open call for new maintainers at this point but you can refer to our [contributing guide](https://contributing.ropensci.org/) for finding ways to get involved!
+As the maintainer of an rOpenSci package, feel free to contact us on Slack or email `info@ropensci.org` to get your call for maintainer featured in the next newsletter. -->
```
[the remark this answers is not in the document]
Comment (maelle) [reply content/blog/2021-08-31-ropensci-news-digest-august-2021/index.md:161]: done, thanks

**Comment by mpadge** [comment]
@steffilazerte @maelle I've added the link to my talk, as as for yours Steffi, it's a direct link to an `mp4`, and not to youtube or whatever. Let me know if there's anything else you would like :smile:

**Pull request merged**